src/intersections/triangles.jl

```diff
@@ -152,39 +152,39 @@ function intersecttype(f::Function, r::Ray{3,T}, t::Triangle{3,T}) where {T}
   det = e₁ ⋅ p
 
   # keep det > 0, modify T accordingly
-  if det > atol(T)
+  if det > atol(T) * oneunit(T)^2
```

Comment:
You seem to be manually fixing all occurrences of atol(T) with a oneunit(T)^2 factor. This is not good for maintenance. If a fix is necessary in multiple lines of  the code, we need to define a new variable like threshold = atol(T)^2 and use it everywhere. Consistent and clean code is super important in a package that large.

Reply:
@juliohm Ammmm. You are right. Patches scattered in the package are not conducive to maintenance. Maybe we should think of a way on the atol function. For example, det> atol(det) or det> atol(typeof)det)) is better than det> atol(T) * oneunit(T)^2. If we write this way, we don't have to multiply by a factor manually.